src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main() {
+        Machine[] machines = new Machine[]{new Bulldozer(),new Truck(),new Excavator()};
```

Comment:
Please, stick to the previous mentor's comment and fix style issues:
```suggestion
    Machine[] machines = new Machine[] {new Bulldozer(), new Truck(), new Excavator()};
```